### PR TITLE
Remove CodeMarkers from Runtime and add unit tests to ToolboxItem and ComponentDesigner #714

### PR DESCRIPTION
**Pull request by Epica3055**
Related to https://github.com/dotnet/winforms/issues/714

## Proposed changes

-  Remove codeMarkers. 
-  Add Verbs,GetService,PostFilterProperties unitTests

## Test methodology 

- 
- Unit Tests
- 

## Test environment(s) 

- 8.0.0-preview.7.23360.15

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9483)

### CODE REVIEW

File: src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs [hunk 1]
```diff
@@ -38,6 +38,60 @@ public void ComponentDesigner_Children_GetDefault_ReturnsExpected()
         Assert.Empty(treeDesigner.Children);
     }
 
+    [Fact]
+    public void ComponentDesigner_GetService_ReturnsExpected()
+    {
+        using var designer = new ComponentDesigner();
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs:44]:
```suggestion
        using ComponentDesigner designer = new();
```

**Review by dreddy-work — CHANGES_REQUESTED** [review]
added suggestion.

**Review by Tanya-Solyanik — COMMENTED** [review]
Looks good, I added only a minor comment.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank  you!

**Review by dmitrii-drobotov — DISMISSED** [review]

File: src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs [hunk 2]
```diff
@@ -2067,7 +2122,7 @@ public void ComponentDesigner_Initialize_NonRootComponent_Success(object host, o
     public void ComponentDesigner_Initialize_NullInheritanceAttribute_Success()
     {
         using var designer = new CustomInheritanceAttributeComponentDesigner(null);
-        using var component = new Component();
+        using Component component = new();
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs:2125]: why not on line 2124?

File: src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs [hunk 3]
```diff
@@ -2025,7 +2080,7 @@ public static IEnumerable<object[]> Initialize_NonRootComponent_TestData()
     [MemberData(nameof(Initialize_NonRootComponent_TestData))]
     public void ComponentDesigner_Initialize_NonRootComponent_Success(object host, object componentChangeService)
     {
-        using var designer = new ComponentDesigner();
+        using ComponentDesigner designer = new();
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs:2083]: why not for mocksite below?

File: src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs [hunk 4]
```diff
@@ -2263,13 +2318,13 @@ public void ComponentDesigner_PreFilterProperties_WithComponentWithKey_Success()
     [Fact]
     public void ComponentDesigner_PreFilterProperties_WithIPersistComponentSettingsComponentWithKey_Success()
     {
-        using var designer = new SubComponentDesigner();
+        using SubComponentDesigner designer = new();
         PropertyDescriptor descriptor = TypeDescriptor.GetProperties(typeof(CustomComponent))[0];
         var properties = new Dictionary<string, PropertyDescriptor>
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/ComponentDesignerTests.cs:2323]: same here.

File: src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/DesignerVerbCollectionTests.cs [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    {
+        yield return new object[] { Array.Empty<DesignerVerb>() };
+        yield return new object[] { new DesignerVerb[] { new DesignerVerb(null, null), null } };
+    }
+
+    [Theory]
+    [MemberData(nameof(Ctor_DesignerVerbArray_TestData))]
+    public void DesignerVerbCollection_Ctor_DesignerVerbArray(DesignerVerb[] value)
+    {
+        var collection = new DesignerVerbCollection(value);
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/DesignerVerbCollectionTests.cs:26]: here.

File: src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/DesignerVerbCollectionTests.cs [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+    [Fact]
+    public void DesignerVerbCollection_Ctor_NullValue_ThrowsArgumentNullException()
+    {
+        Assert.Throws<ArgumentNullException>("value", () => new DesignerVerbCollection(null));
+    }
+
+    [Fact]
+    public void DesignerVerbCollection_Add_DesignerVerb_Success()
+    {
+        var collection = new DesignerVerbCollection();
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Design/tests/UnitTests/System/ComponentModel/Design/DesignerVerbCollectionTests.cs:39]: in this file

**Review by dreddy-work — CHANGES_REQUESTED** [review]
we are enforcing pattern for object initialization with type declaration on left side of expression wherever possible.

i.e :` Object object=  new();`

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you for applying this pattern to the code base!

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**